src/app/package-json.js

```diff
@@ -268,6 +272,8 @@ const packageJSON = (current, context) => {
 
   // Support the documentation tooling option.
   if (context.docs) {
+    result.files.push('docs');
```

Comment:
Having the `src` in there allows users who want to do their own compilation the option to do so (most commonly webpack users). Having both `src` and `tests` in the tarball also makes it possible to completely recover a repo from the npm package if necessary.